src/Core/Traits/PostValueRestrictions.php

```diff
@@ -15,12 +15,12 @@
 
 use Ushahidi\Core\Entity\User;
 use Ushahidi\Core\Entity\Post;
+use Ushahidi\Core\Entity\Permission;
 use Ushahidi\Core\Entity\FormRepository;
 
 trait PostValueRestrictions
```

Comment:
I'm not at all sure this, and the classes which use it are set up right. Or the naming is just weird. I might need to pair up to review it.